### PR TITLE
Clarification of contribution guidelines

### PR DESCRIPTION
**Pull request by tkowark**
## Description

To avoid disappointment for community members, this PR clarifies the contribution guidelines to the cwa-app-android repository. In particular, two aspects are tackled:

* Feature requests shall now be created in the wishlist repository to avoid closing of tickets because of issues that will not likely be implemented or fixed in the near future
* By removing ambiguity as to how community member shall contribute through PRs, we aim to avoid unnecessary work on your end. In short, the guideline is: PRs shall only be created if an issue was opened up beforehand and this issue was assigned by one of the maintainers.

### CODE REVIEW

**Comment by tkowark** [comment]
Of course, this only affects new PRs

**Review by marcmuschko — DISMISSED** [review]
As discussed, existing community PRs are excluded from the newly added guidelines, we will review + accept / decline / close and comment these.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Before submitting, please take the time to check the points below and provide so
 * [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
 * [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
 * [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
-* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
+* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
```
Comment (MichaelPFrey) [inline .github/pull_request_template.md:11]: Does this also apply to minor pull requests like fixing typos and grammar in documentation and comments?

People can not self-assignee to an issue, so fixing one typo will be come an involved process of
- opening a bug issue (what else is a typo?), showing the erroneous line of code/text
- waiting for getting the issue assigned
- fixing the typo
- opening a pull request
or
- opening a bug issue (what else is a typo?), showing the erroneous line of code/text
- a maintainer fixing the typo himself and then closing the issue

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,33 +4,37 @@
...
+We appreciate your courtesy of avoiding political questions here. Issues which are not related to the project itself will be closed by our community managers.
 
 ## Engaging in Our Project
 
 We use GitHub to manage reviews of pull requests.
 
 * If you are a new contributor, see: [Steps to Contribute](#steps-to-contribute)
 
-* If you have a trivial fix or improvement, go ahead and create a pull request, addressing (with `@...`) a suitable maintainer of this repository (see [CODEOWNERS](CODEOWNERS) of the repository you want to contribute to) in the description of the pull request.
+* Before opening a new Pull Request, create an issue that describes the problem you would like to solve or the code that should be enhanced. Please note that you are willing to work on that issue.
```
Comment (MichaelPFrey) [inline CONTRIBUTING.md:17]: Something in the logic here is backward.

If you want people to open an issue before doing any work, then say so.
```suggestion
* Before implementing your change, create an issue that describes the problem you would like to solve or the code that should be enhanced. Please note that you are willing to work on that issue.
```
or
```suggestion
* Before opening a new Pull Request, create an issue that describes the problem you would like to solve or the code that should be enhanced. Please note that you are willing to work on that issue. If you have already started, you can link to your fork.
```
At the point at which people can/could/would open a pull request, they have an implementation. If they have an implementation, they might as well link to their fork, as seeing the practical implementation might swing in favor or against a certain idea.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,4 +76,4 @@ The following rule governs documentation contributions:
 
 * When creating an issue, try using one of our issue templates which already contain some guidelines on which content is expected to process the issue most efficiently. If no template applies, you can of course also create an issue from scratch.
 
-* Please apply one or more applicable [labels](https://github.com/corona-warn-app/cwa-app-android/labels) to your issue so that all community members are able to cluster the issues better.
\ No newline at end of file
+* Please apply one or more applicable [labels](https://github.com/corona-warn-app/cwa-app-android/labels) to your issue so that all community members are able to cluster the issues better.
```
Comment (MichaelPFrey) [inline CONTRIBUTING.md:79]: Are you aware that only project members can add and remove labels on github?
Asking general github users/"the community" to apply labels is therefor a bit pointless.

File: .github/pull_request_template.md (hunk 1 above)
Comment (tkowark) [reply .github/pull_request_template.md:11]: Agreed, I will change that accordingly

File: CONTRIBUTING.md (hunk 3 above)
Comment (tkowark) [reply CONTRIBUTING.md:79]: Thanks for spotting that. We'll check this sentence in the other repos, too.

**Review by SebastianWolf-SAP — APPROVED** [review]

**Pull request merged**